src/applications/disability-benefits/all-claims/pages/ptsdWalkthroughChoice781a.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import { PtsdNameTitle } from '../content/ptsdClassification';
+import {
+  PtsdUploadChoiceDescription,
+  UploadPtsdDescription,
+} from '../content/ptsdWalkthroughChoice';
+
+export const uiSchema = {
+  'ui:title': ({ formData }) => (
+    <PtsdNameTitle formData={formData} formType="781a" />
+  ),
+  'ui:description': ({ formData }) => (
+    <UploadPtsdDescription formData={formData} formType="781a" />
+  ),
+  'view:uploadPtsdChoice': {
+    'ui:title': ' ',
+    'ui:widget': 'radio',
+    'ui:options': {
+      labels: {
+        answerQuestions: 'I want to answer questions',
+        upload: 'I want to upload VA Form 21-0781a',
+      },
+    },
+  },
+  'view:uploadPtsdChoiceHelp': {
+    'ui:description': <PtsdUploadChoiceDescription formType="781a" />,
+  },
+};
+
+export const schema = {
+  type: 'object',
+  properties: {
+    'view:uploadPtsdChoice': {
```

Comment:
Should this property be required? If the veteran doesn't answer this question, they'll be moved past the follow-up pages and into the unemployment questions.

Reply:
I think that's the idea. Because the questions about PTSD may trigger an event (or...I'm not sure what the right way of saying that is), we're not requiring them to answer any questions and can submit an essentially blank form.

Does that sound about right, @aaronjlech?